cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature

```diff
@@ -113,7 +156,68 @@ Feature: Discovery.launch_ColdLaunch
         Then 'Firebolt' platform responds with '<Validation_Key_For_Parameters_Initialization_With_Intent>'
         And 'Firebolt' platform responds with 'foreground for lifecycle state'
 
-        Examples:
-            | Scenario              | Discovery_Launch_Key                   | Call_Parameters_Initialization_With_Context_Key   | Validation_Key_For_Parameters_Initialization_With_Intent |
-            | Passing random string | launch app with null intent source     | get initialization parameters for null intent     | nullintent source for initialization parameters          |
-            | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
\ No newline at end of file
+    Examples:
+        | Scenario              | Discovery_Launch_Key                   | Call_Parameters_Initialization_With_Context_Key   | Validation_Key_For_Parameters_Initialization_With_Intent |
+        | Passing random string | launch app with null intent source     | get initialization parameters for null intent     | nullintent source for initialization parameters          |
+        | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
+
+    # Below two scenarios are purely added for sdk certification testing and sign off due to limitation in testing discovery launch testcases with current support
```

Comment:
below test cases to be removed before merge

Reply:
done